### PR TITLE
Update headless Chrome docs

### PR DESCRIPTION
**Pull request by paulirish**
I landed #2554 but then reverted.

This PR restores eric's commit so we can discuss headless vs xvfb headless.

### CODE REVIEW

File: docs/headless-chrome.md [hunk 1]
```diff
@@ -1,28 +1,62 @@
-# Running headless Chrome for Lighthouse
+# Running Lighthouse using headless Chrome
 
-The headless_shell still has a few bugs to work out. Until then, Chrome + xvfb is a stable solution.
```
Comment (patrickhulce) [inline docs/headless-chrome.md]: I think we should keep the reference and instructions to xvfb until we're confident in headless parity, maybe just split into 2 sections and add a caveat to the headless instructions with a link to the section lower down on xvfb?

Comment (ebidel) [inline docs/headless-chrome.md]: done

**Comment by ebidel** [comment]
ptal

**Comment by benschwarz** [comment]
FYI, I'm not rushing into headless_shell either, seems too early still

File: docs/headless-chrome.md [hunk 2]
```diff
@@ -12,17 +15,86 @@ sudo apt-get install -y nodejs
...
+Run it:
+
 ```sh
 export DISPLAY=:1.5
 TMP_PROFILE_DIR=$(mktemp -d -t lighthouse.XXXXXXXXXX)
 
 # start up chromium inside xvfb
-xvfb-run --server-args='-screen 0, 1024x768x16' chromium-browser --user-data-dir=$TMP_PROFILE_DIR --start-maximized --no-first-run --remote-debugging-port=9222 "about:blank"
+xvfb-run --server-args='-screen 0, 1024x768x16' \
+    chromium-browser --user-data-dir=$TMP_PROFILE_DIR
```
Comment (brendankenny) [inline docs/headless-chrome.md]: switch to `chrome-launcher`?

Comment (ebidel) [inline docs/headless-chrome.md]: i'd have to test it...and ugh!

**Pull request merged**